### PR TITLE
Update read me with recent updates

### PR DESCRIPTION
**Pull request by NBParis**
It is now possible to add custom tags to the logs thanks to the `DD_TAGS` environment variable.

This needed to be included in the read.me

### CODE REVIEW

File: Log/README.md [hunk 1]
```diff
@@ -62,6 +62,13 @@ metadata = {
 }
 ```
 
+- **Custom Tags** 
+
+You have two options to optionnally add custom tags to the logs:
```
Comment (l0k0ms) [inline Log/README.md]: r/`You have two options to optionnally add custom tags to the logs:`/ `You have two options to optionally add custom tags to your logs:`

File: Log/README.md [hunk 2]
```diff
@@ -62,6 +62,13 @@ metadata = {
...
+
+- Manually by editing the lambda code [there](https://github.com/DataDog/dd-aws-lambda-functions/blob/master/Log/lambda_function.py#L79)
+- With the `DD_TAGS` environment variable (tag must be comma separated).
```
Comment (l0k0ms) [inline Log/README.md]: r/```With the `DD_TAGS` environment variable (tag must be comma separated).```/ ```Automatically with the `DD_TAGS` environment variable (tags must be a comma-separated list of strings).```

File: Log/README.md (hunk 1 above)
Comment (NBParis) [reply Log/README.md]: @l0k0ms what about adding `(Optional)` in the title and then 
`You have two configuration possibilities to add custom tags to your logs:`

**Comment by NBParis** [comment]
@l0k0ms updated with your comments

**Pull request merged**